src/Providers/ProvidersPage.tsx

```diff
@@ -0,0 +1,218 @@
+import React, { useMemo, useState } from 'react';
+import { useTranslation } from 'src/internal/i18n';
+import { ProviderResource } from 'src/internal/k8s';
+
+import { MOCK_CLUSTER_PROVIDERS } from '@app/queries/mocks/providers.mock';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Button,
+  Level,
+  LevelItem,
+  PageSection,
+  Title,
+  Toolbar,
+  ToolbarContent,
+  ToolbarToggleGroup,
+} from '@patternfly/react-core';
+import { FilterIcon } from '@patternfly/react-icons';
+
+import AttributeValueFilter from './components/AttributeValueFilter';
+import EnumFilter from './components/EnumFilter';
+import FreetextFilter from './components/FreetextFilter';
+import { ManageColumnsToolbar } from './components/ManageColumnsToolbar';
+import PrimaryFilters from './components/PrimaryFilters';
+import ProviderRow from './components/ProviderRow';
+import { createMetaMatcher, Field } from './components/shared';
+import { NAME, NAMESPACE, READY, TYPE, URL } from './components/shared';
+import TableView from './components/TableView';
+
+const isMock = process.env.DATA_SOURCE === 'mock';
+
+const useProviders = ({ kind, namespace }) => {
+  const [providers, loaded, error] = isMock
+    ? [MOCK_CLUSTER_PROVIDERS, true, false]
+    : useK8sWatchResource<ProviderResource[]>({
+        kind,
+        isList: true,
+        namespaced: true,
+        namespace,
+      });
+
+  // const inventoryProvidersQuery = useInventoryProvidersQuery();
+  // providers.map(p => enhanceWithInventory(inventoryProvidersQuery))
+
+  // const allErrorTitles = [
+  //   'Cannot load providers from cluster API',
+  //   'Cannot load providers from inventory API',
+  // ];
+
+  return [providers, loaded, error];
+};
+
+const defaultFields: Field[] = [
+  {
+    id: NAME,
+    tKey: 'plugin__forklift-console-plugin~Name',
+    isVisible: true,
+    isIdentity: true,
+    filter: {
+      type: 'freetext',
+      placeholderKey: 'plugin__forklift-console-plugin~FilterByName',
+    },
+    sortable: true,
+    toValue: (provider) => provider?.metadata?.name ?? '',
+  },
+  {
+    id: NAMESPACE,
+    tKey: 'plugin__forklift-console-plugin~Namespace',
+    isVisible: true,
+    isIdentity: true,
+    filter: {
+      type: 'freetext',
+      placeholderKey: 'plugin__forklift-console-plugin~FilterByNamespace',
+    },
+    sortable: true,
+    toValue: (provider) => provider?.metadata?.namespace ?? '',
+  },
+  {
+    id: READY,
+    tKey: 'plugin__forklift-console-plugin~Ready',
```

Comment:
This is actually the "condition" ready, maybe "Condition" with value "Ready/Not Ready" is better ?

Reply:
1.  CLI uses label `READY` and values true/false (and perhaps unknown). Right now we use the same convention. 
2.  At least in English label `Ready` and `Not Ready` look similar - it would be nice to have some visually different label 
3. actually we are not showing all values - k8s condition may have 3 states: true, false or unknown - see [this post](https://github.com/kubernetes/kubernetes/issues/7856#issuecomment-323196033)

Reply:
nice find with the post,
so we leave the title "Ready" like the CLI, makes sense to me,
I have concern about the red icon when not ready, a provider can be not ready but ok, for example if it's still getting inventory, we should think of better content when the "Ready" value is "fails" to represent that it's not necceraly a "bad" thing, and a user should see all conditions to know what "false" mean.

Reply:
In case when `READY` condition is not explicit `false` we report `Unknown` (see the screenshots section).